presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -341,6 +344,56 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+S3SelectPushdown
+^^^^^^^^^^^^^^^^
+S3SelectPushdown is a feature of Amazon S3 Select that enables Presto users to push down projection (SELECT)
```

Comment:
Would it make sense to link to a page that has an overview of Amazon S3 Select?

Reply:
I think it would add value for users who are not familiar with the service. Added the link.